File: app_torneig_flutter/lib/models/equip_partit_dto.dart
class EquipPartitDTO {
  int? id;
  String? nom;
  String? grup;
  String? imatge;
  bool? esGuanyador;

  EquipPartitDTO({
    required this.id,
    this.nom,
    this.grup,
    this.imatge,
    this.esGuanyador,
  });

  EquipPartitDTO.fromJSON(Map<String, dynamic> objecteJSON) {
    id = objecteJSON['idEquip'] ?? "";
    nom = objecteJSON['nom'] ?? "";
    grup = objecteJSON['nomGrup'] ?? "";
    imatge = objecteJSON['imatge'] ?? "assets/images/defecte.png";
    esGuanyador = objecteJSON['esGuanyador'];
  }
}

File: Torneig_Mort_Spring/src/main/java/org/eljust/DTO/EquipPartitDTO.java
package org.eljust.DTO;

import lombok.Getter;
import lombok.Setter;

/**
 * Aquet DTO Equip, el fem servir per a poder gestionar els equips d'un partit,
 * ja que asi tenim equip local i equip visitant. El gastem a PartitDTO.
 */
@Setter
@Getter
public class EquipPartitDTO {
	private Long idEquip;
	private String nom;
	private String imatge;
	private boolean esGuanyador;
	private String nomGrup;

}
